landoapi/spec/swagger.yml

```diff
@@ -186,6 +186,41 @@ paths:
           schema:
             allOf:
               - $ref: '#/definitions/Error'
+  /submitSanitizedCommitMessage:
```

Comment:
This also kicks off the actual request for `sec-approval`. I wonder if something like `/requestSecApproval` would be more appropriate (also, shorter, which would be nice :D)